src/main/java/com/rocketden/main/controller/v1/UserController.java

```diff
@@ -25,4 +27,9 @@ public UserController(UserService service) {
     public ResponseEntity<UserDto> createUser(@RequestBody CreateUserRequest request) {
         return new ResponseEntity<>(service.createUser(request), HttpStatus.CREATED);
     }
+
+    @DeleteMapping("/user")
+    public ResponseEntity<UserDto> deleteUser(@RequestBody DeleteUserRequest request) {
+        return new ResponseEntity<>(service.deleteUser(request), HttpStatus.ACCEPTED);
```

Comment:
Am I correct in assuming that this code only goes through if the user is successfully deleted, and otherwise, it does not return the `HttpStatus.ACCEPTED` but instead throws an exception? If that's the case, should this return `HttpStatus.OK` based on [this article](https://briancaos.wordpress.com/2019/06/19/200-ok-vs-202-accepted-whats-the-difference/#:~:text=200%20OK%20means%20that%20the,the%20service%20will%20now%20start.)? (If so, you'll want to update the tests accordingly as well.)

Reply:
Yes to the first part. Sure I can switch it to an OK status; I mainly used ACCEPTED because I saw others use it with @deletemapping.